src/vivarium_csu_ltbi/tools/builder.py

```diff
@@ -0,0 +1,93 @@
+from pathlib import Path
+from loguru import logger
+
+
+from gbd_mapping import causes
+from vivarium_public_health.dataset_manager import Artifact, EntityKey, get_location_term
+from vivarium_inputs.data_artifact.utilities import split_interval
+from vivarium_inputs import get_measure, utilities, globals, utility_data, core
+
+DEFAULT_PATH = '/ihme/homes/kjells/artifacts'
```

Comment:
let's set the default where it will live ultimately - the artifacts repo under share costeffect, in a vivarium_csu_ltbi folder

Reply:
OK, done.